GundaGunde/GG00044.xml

```diff
@@ -0,0 +1,1356 @@
+                          </decoNote>
+                          
+                          <decoNote type="miniature" xml:id="d28">
+                              <locus target="#60vb"/>
+                             <locus target="#61r"/>
+                              <desc><ref type="authFile" corresp="AT1135MMSalusi"/></desc>
+                          </decoNote>
+                          
+                          <decoNote type="miniature" xml:id="d29">
+                              <locus target="#62vb"/>
+                              <locus target="#63r"/>
+                              <desc><ref type="authFile" corresp="AT1136MMCannibal"/></desc>
+                          </decoNote>
+                          
+                          <decoNote type="miniature" xml:id="d30">
+                              <locus target="#64v"/>
+                              <desc><ref type="authFile" corresp="AT1137MMThief"/></desc>
+                          </decoNote>
+                          
+                          <decoNote type="miniature" xml:id="d31">
+                              <locus target="#65vb"/>
+                             <locus target="#66r"/>
+                              <desc><ref type="authFile" corresp="AT1138MMFlood"/> </desc>
+                          </decoNote>
+                          
+                          <decoNote type="miniature" xml:id="d32">
+                             <locus target="#67v"/>
+                              <desc><ref type="authFile" corresp="AT1139MMDog"/> </desc>
+                          </decoNote>
+                        <decoNote type="miniature" xml:id="d33">
+                                 
+                        <locus from="87v" to="88v"/>
+                        <desc><ref type="authFile" corresp="AT1036MercyCovenant"/></desc>
+                     </decoNote>
+                     </decoDesc>
+                     
+            
+         </physDesc>
+               </msPart>
+               <msPart xml:id="p2">
+                  <msIdentifier/>
+                  <msContents>
+                     <summary/>
+                     
+                     <msItem xml:id="p2_i1">
+                        <locus from="92ra" to="98vb"/>
+                        <title type="complete" ref="LIT5437MiracleCommandments"/>
+                        <textLang mainLang="gez"/>
+                     </msItem>
+                     </msContents>
+                     <physDesc>
+                        <objectDesc form="Codex">
+                           <supportDesc>
+                              <support>
+                                 <material key="parchment"/>
+                              </support>
+                              <extent>
+                                 <measure unit="leaf">6</measure>
+                                 <locus from="1ra" to="5ra"/>
```

Comment:
the text this codicological unit contains is on     `<locus from="92ra" to="98vb"/>`, but the unit is  noted to be on  `<locus from="1ra" to="5ra"/>` - I think one of them must be wrong.